tests/js/spec/views/dashboardsV2/gridLayout/dashboard.spec.tsx

```diff
@@ -4,6 +4,7 @@ import {
   mountWithTheme as rtlMountWithTheme,
```

Comment:
We do not normally rename named exports. Should we just use `mountWithTheme`?

Reply:
This was my bad. We haven't fully converted tests to RTL so instead of making a separate file with RTL tests, the new ones I added were using this `rtlMountWithTheme`

Reply:
Hey Priscila, some of our test files haven't been fully converted to RTL yet, so we've got files with a mix of enzyme and RTL. This naming is so we don't clash on importing `mountWithTheme` for now. We'll eventually update all the enzyme tests to RTL in the future and undo the rename.